sandBox/FirstSprint/TaskLoops.cs

```diff
@@ -0,0 +1,108 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.Concurrent;
+using System.Linq;
+using System.Text;
+
+namespace FirstSprint
+{
+    class TaskLoops
+    {
+        private string[] GetRandomStringArray()
+        {
+            int counter = 0;
+            string alphabet = "QWERTYUIOPASDFGHJKLZXCVBNMqwertyuiopasdfghjklzxcvbnm";
```

Comment:
You can use ASCII range [065;122] instead of constant variables that are not marked as constant...